phalcon/mvc/router.zep

```diff
@@ -574,7 +574,7 @@ class Router implements RouterInterface
 			 */
 			if fetch paramsStr, parts["params"] {
 				let strParams = substr(paramsStr, 1);
-				if strParams {
+				if strlen(strParams) > 0 {
```

Comment:
But substr will return "false" when paramsStr is only 1 character length. So pull request #3280 is better...